### PR TITLE
ci: introduce skywalking-eyes

### PR DESCRIPTION
**Pull request by fgksgf**
Introduce apache/skywalking-eyes to check the license header.

### CODE REVIEW

File: .github/workflows/license.yaml [hunk 1]
```diff
@@ -1,38 +0,0 @@
...
-    runs-on: ubuntu-latest
-    steps:
-      - name: Set up Python 3.7
-        uses: actions/setup-python@v2
-        with:
-          python-version: 3.7
-      - name: Check out code into the Go module directory
-        uses: actions/checkout@v2
-      - name: Check license header
-        run: make license
```
Comment (kezhenxu94) [inline .github/workflows/license.yaml:37]: I think the original `make license` can be remove or refactored to use license eye, (p.s. we have Docker image for license-eye)

File: .licenserc.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+header:
+  license:
+    spdx-id: Apache-2.0
+    copyright-owner: Apache Software Foundation
```
Comment (kezhenxu94) [inline .licenserc.yaml:20]: This repo is not in ASF, so the copyright owner should not be ASF

File: .github/workflows/license.yaml (hunk 1 above)
Comment (fgksgf) [reply .github/workflows/license.yaml:37]: Ok. 

By the way, should I remove the python script in `tools/`, which is used to check license headers?

File: .licenserc.yaml (hunk 2 above)
Comment (fgksgf) [reply .licenserc.yaml:20]: copy-paste issue, should be `SkyAPM org` right ?

File: .github/workflows/license.yaml (hunk 1 above)
Comment (kezhenxu94) [reply .github/workflows/license.yaml:37]: > Ok.
> 
> By the way, should I remove the python script in `tools/`, which is used to check license headers?

I suggest to remove it.

File: .licenserc.yaml (hunk 2 above)
Comment (kezhenxu94) [reply .licenserc.yaml:20]: > copy-paste issue, should be `SkyAPM org` right ?

Yes, you can test it locally

**Comment by kezhenxu94** [comment]
Also, the existing license headers are the one specific to ASF projects, so we'd better fix them

**Comment by wu-sheng** [comment]
> License Expected — Waiting for status to be reported

Pending check is missing. I think you need to keep it or update .asf.yml first in another PR.

**Comment by kezhenxu94** [comment]
> > License Expected — Waiting for status to be reported
> 
> Pending check is missing. I think you need to keep it or update .asf.yml first in another PR.

This is not ASF project so we don't have `.asf.yml` 😉 Just remove that `require`

**Review by kezhenxu94 — APPROVED** [review]

**Pull request merged**